stories/Tag.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { Story } from '@storybook/react';
+
+import { Tag } from '../src/components/atoms';
+import { Props as TagProps } from '../src/components/atoms/Tag/Tag';
+
+export default {
+  title: 'Tag',
+  component: Tag
+};
+
+const tagText = '0 shares are available to sell';
+
+
+const Template: Story<TagProps> = (args) => <Tag {...args}>
+  <div className="absolute-center">
+    {tagText}
+  </div>
+</Tag>;
+
+
+export const Warning = Template.bind({});
+
+export const Error = Template.bind({});
```

Comment:
What about Info?

Reply:
Currently, info is not deployed once done will update story for it

Reply:
added story for it in advance